fdbserver/BackupWorker.actor.cpp

```diff
@@ -878,25 +878,19 @@ ACTOR Future<Void> uploadData(BackupData* self) {
 		int lastVersionIndex = 0;
 		Version lastVersion = invalidVersion;
 
-		if (self->messages.empty()) {
-			// Even though messages is empty, we still want to advance popVersion.
-			if (!self->endVersion.present()) {
-				popVersion = std::max(popVersion, self->minKnownCommittedVersion);
+		for (auto& message : self->messages) {
```

Comment:
do we still want to advance `popVersion` when messages is empty?

Reply:
popVersion will be advanced anyway in line 896 or 899.